Packs/CortexXDR/Integrations/CortexXDRIR/README.md

```diff
@@ -157,6 +157,7 @@ To setup the mirroring follow these instructions:
   - The default playbook of the *Cortex XDR Incident* incident type is not *Cortex XDR Incident Sync*, change it to a 
      different playbook that does not use `XDRSyncScript`.
   - The XDR integration instance incoming mapper is set to `Cortex XDR - Incoming Mapper` and the outgoing mapper is set to `Cortex XDR - Outgoing Mapper`.
+  - Mirroring impacts only incidents that were fetched after the mirroring was enabled for this instance. If incidents were fetched with the incorrect mapper, changing the mapper will not affect them. This can be resolved by creating a new instance with the correct mapper.
```

Comment:
```suggestion
  - Mirroring impacts only incidents that were fetched after the mirroring was enabled for this instance. If incidents were fetched with the incorrect mapper, changing the mapper will not affect them. This can be resolved by creating a new integration instance with the correct mapper. New incidents will be created and the old ones will no longer be relevant.
```